### PR TITLE
[hi] Add content/hi/docs/setup/production-environment/_index.md

### PR DESCRIPTION
**Pull request by keshavcodex**
Localizes

- content/hi/docs/setup/production-environment/_index.md 

This PR is in continuation of hi localization effort: #29353

/language hi

### CODE REVIEW

**Comment by sftim** [comment]
Hi @keshavcodex 

If you have a moment, would you be willing to revise the PR description? It should be a summary of the changes you are making.

**Comment by keshavcodex** [comment]
> Hi @keshavcodex
> 
> If you have a moment, would you be willing to revise the PR description? It should be a summary of the changes you are making.

i made changes please check if that is correct.

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/setup/production-environment/_index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+- *DNS मांग के लिए तैयार करें*: यदि आप वर्कलोड के बड़े पैमाने पर बढ़ने की उम्मीद करते हैं, तो आपकी DNS सेवा भी बड़े पैमाने पर तैयार होनी चाहिए।
+[क्लस्टर में DNS सेवा को ऑटोस्केल करना](/docs/tasks/administer-cluster/dns-horizontal-autoscaling/) देखें।
+- *अतिरिक्त सेवा खाते बनाएं*: उपयोगकर्ता खाते यह निर्धारित करते हैं कि उपयोगकर्ता क्लस्टर पर क्या कर सकते हैं, जबकि एक सेवा खाता किसी विशेष नेमस्पेस्स के भीतर पॉड एक्सेस को परिभाषित करता है। डिफ़ॉल्ट रूप से, एक पॉड अपने नेमस्पेस्स से डिफ़ॉल्ट सेवा खाता लेता है।
+नया सेवा खाता बनाने के बारे में जानकारी के लिए [सेवा खातों का प्रबंधन](/docs/reference/access-authn-authz/service-accounts-admin/) देखें।
+उदाहरण के लिए, आप शायद यह करना चाहें:
+  - ऐसे रहस्य जोड़ें जिनका उपयोग पॉड किसी विशेष कंटेनर रजिस्ट्री से इमेज खींचने के लिए कर सकता है। उदाहरण के लिए [पॉड्स के लिए सेवा खाते कॉन्फ़िगर करें](/docs/tasks/configure-pod-container/configure-service-account/) देखें।
+  - किसी सेवा खाते में RBAC अनुमतियाँ असाइन करें। विवरण के लिए [सेवा खाता अनुमतियां](/docs/reference/access-authn-authz/rbac/#service-account-permissions) देखें।
+
+## {{% heading "whatsnext" %}}
```
Comment (jayesh-srivastava) [inline content/hi/docs/setup/production-environment/_index.md:125]: I guess this should be written as `आगे क्या है`

File: content/hi/docs/setup/production-environment/_index.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+- *व्यवस्थित कण्ट्रोल प्लेन*: प्रदाता को क्लस्टर के कण्ट्रोल प्लेन के पैमाने और उपलब्धता का प्रबंधन करने दें, साथ ही पैच और अपग्रेड को भी संभालें।
+- *व्यवस्थित वर्कर नोड्स*: अपनी आवश्यकताओं को पूरा करने के लिए नोड्स के पूल को कॉन्फ़िगर करें, फिर प्रदाता सुनिश्चित करता है कि वे नोड उपलब्ध हैं और जरूरत पड़ने पर अपग्रेड को लागू करने के लिए तैयार हैं।
+- *एकीकरण*: ऐसे प्रदाता हैं जो कुबेरनेट्स को अन्य सेवाओं के साथ एकीकृत करते हैं जिनकी आपको आवश्यकता हो सकती है, जैसे भंडारण, कंटेनर रजिस्ट्रियां, प्रमाणीकरण तरीके, और विकास उपकरण।
+
+चाहे आप कुबेरनेट्स क्लस्टर का निर्माण स्वयं करें या भागीदारों के साथ काम करें, अपनी आवश्यकताओं का मूल्यांकन करने के लिए निम्नलिखित अनुभागों की समीक्षा करें क्योंकि वे आपके क्लस्टर से संबंधित हैं *कण्ट्रोल प्लेन*, *वर्कर नोड्स*, *यूज़र एक्सेस*, और *वर्कलोड रिसोर्सस*.
+
+## उत्पादन क्लस्टर सेटअप
+
+उत्पादन-गुणवत्ता वाले कुबेरनेट्स क्लस्टर में, कण्ट्रोल प्लेन क्लस्टर को उन सेवाओं से प्रबंधित करता है जिन्हें विभिन्न तरीकों से कई कंप्यूटरों में फैलाया जा सकता है। हालाँकि, प्रत्येक कार्यकर्ता नोड एक एकल इकाई का प्रतिनिधित्व करता है जिसे कुबेरनेट्स पॉड चलाने के लिए कॉन्फ़िगर किया गया है।
```
Comment (jayesh-srivastava) [inline content/hi/docs/setup/production-environment/_index.md:47]: Shouldn't `कार्यकर्ता` be written as `वर्कर` itself as it is a technical term? @anubha-v-ardhan

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

File: content/hi/docs/setup/production-environment/_index.md (hunk 2 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/setup/production-environment/_index.md:47]: yep

**Comment by anubha-v-ardhan** [comment]
Hey @keshavcodex Are you active and working with this PR?

File: content/hi/docs/setup/production-environment/_index.md (hunk 1 above)
Comment (keshavcodex) [reply content/hi/docs/setup/production-environment/_index.md:125]: sorry i didn't get it, which word do i need to change?

**Comment by keshavcodex** [comment]
> Hey @keshavcodex Are you active and working with this PR?

yes, i will do this, i was inactive because of my exams.

File: content/hi/docs/setup/production-environment/_index.md (hunk 1 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/setup/production-environment/_index.md]: @jayesh-srivastava This shouldn't be translated. These inside `{{}}` are just placeholders. Hugo (Kubernetes's static site generator) will do its job and replace them with terms from the hi.toml file

File: content/hi/docs/setup/production-environment/_index.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+- *DNS मांग के लिए तैयार करें*: यदि आप वर्कलोड के बड़े पैमाने पर बढ़ने की उम्मीद करते हैं, तो आपकी DNS सेवा भी बड़े पैमाने पर तैयार होनी चाहिए।
+[क्लस्टर में DNS सेवा को ऑटोस्केल करना](/docs/tasks/administer-cluster/dns-horizontal-autoscaling/) देखें।
+- *अतिरिक्त सेवा खाते बनाएं*: उपयोगकर्ता खाते यह निर्धारित करते हैं कि उपयोगकर्ता क्लस्टर पर क्या कर सकते हैं, जबकि एक सेवा खाता किसी विशेष नेमस्पेस्स के भीतर पॉड एक्सेस को परिभाषित करता है। डिफ़ॉल्ट रूप से, एक पॉड अपने नेमस्पेस्स से डिफ़ॉल्ट सेवा खाता लेता है।
+नया सेवा खाता बनाने के बारे में जानकारी के लिए [सेवा खातों का प्रबंधन](/docs/reference/access-authn-authz/service-accounts-admin/) देखें।
+उदाहरण के लिए, आप शायद यह करना चाहें:
+  - ऐसे रहस्य जोड़ें जिनका उपयोग पॉड किसी विशेष कंटेनर रजिस्ट्री से इमेज खींचने के लिए कर सकता है। उदाहरण के लिए [पॉड्स के लिए सेवा खाते कॉन्फ़िगर करें](/docs/tasks/configure-pod-container/configure-service-account/) देखें।
+  - किसी सेवा खाते में RBAC अनुमतियाँ असाइन करें। विवरण के लिए [सेवा खाता अनुमतियां](/docs/reference/access-authn-authz/rbac/#service-account-permissions) देखें।
+
+## {{% heading "आगे क्या है" %}}
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/production-environment/_index.md:125]:
```suggestion
## {{% heading "whatsnext" %}}
```

Comment (anubha-v-ardhan) [reply content/hi/docs/setup/production-environment/_index.md:125]: See https://github.com/kubernetes/website/pull/29518#discussion_r785278840

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/setup/production-environment/_index.md (hunk 3 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/setup/production-environment/_index.md:125]: @keshavcodex If you can squash the commits after committing the above suggestions, please do

**Comment by anubha-v-ardhan** [comment]
@keshavcodex Are you working on this PR. If yes please commit the [above suggestion](https://github.com/kubernetes/website/pull/29518#discussion_r785278991), I guess we could merge it then.

**Review by anubha-v-ardhan — APPROVED** [review]
/lgtm
/approve

Thankyou @keshavcodex :)

**Pull request merged**